runtime/oti/vm_api.h

```diff
@@ -1257,6 +1257,16 @@ initializeINLInterception(J9JavaVM *vm);
 void
 initializeInitialMethods(J9JavaVM *vm);
 
+/* ---------------- jfr.cpp ---------------- */
+
+/**
+ * @brief Determine the size of a JFR event
+ * @param jfrEvent the event
+ * @return the size of the JFR event in bytes
+ */
+UDATA
+jfrEventSize(J9JFREvent *jfrEvent);
```

Comment:
This isn't used outside of the buffer iterator functions, so there's no reason to make it public.